conf.py

```diff
@@ -374,7 +374,7 @@
     "druid": {"id": "626140028ccb9938ba3cfde7"},
     "hadoop": {"id": "6261407f887d6e0b8614660c"},
     "hbase": {"id": "62614109992bac3f9a4a24b8"},
-    "hive": {"id": "626140028ccb9938ba3cfde7"},
+    "hive": {"id": "626140806812078a392dceaa"},
```

Comment:
I have no idea what this is about but I'm sure you do :)

Reply:
Yes, the id was wrong :)